src/menu.ts

```diff
@@ -130,6 +135,19 @@ export class Menu implements TeleportInteractiveElement {
         mobileMenu.classList.remove("thq-show");
         mobileMenu.classList.remove("thq-translate-to-default");
       });
+
+      // check if user clicked on a scrollTo link in the mobile menu and close the menu
+      mobileMenu.addEventListener("click", (event) => {
+        const target = event.target as HTMLElement;
+        if (
+          target.tagName === "A" &&
+          target.getAttribute("href")?.startsWith("#")
```

Comment:
Ideally we should be closing for any type of clicks outside the menu, instead of manually checking the tag-names like this here.

Reply:
We are in mobile view, the mobile menu  is 100% width, 100vh...there is no click outside of the menu